src/adapters/MeetingsJSONAdapter.test.js

```diff
@@ -0,0 +1,49 @@
+import meetings from './../data/meetings';
+import MeetingsJSONAdapter from './MeetingsJSONAdapter';
+
+describe('Meetings JSON Adapter Interface', () => {
+  let meetingsJSONAdapter, meetingID;
+
+  beforeEach(() => {
+    [meetingID] = Object.keys(meetings);
+    meetingsJSONAdapter = new MeetingsJSONAdapter(meetings);
+  });
+
+  test('getMeeting() returns an observable', () => {
+    expect(rxjs.isObservable(meetingsJSONAdapter.getMeeting())).toBeTruthy();
+  });
+
+  test('getMeeting() returns a meeting data', (done) => {
+    meetingsJSONAdapter.getMeeting(meetingID).subscribe((data) => {
+      expect(data).toEqual(meetings[meetingID]);
+      done();
+    });
+  });
+
+  test('getMeeting() throws a proper error message', (done) => {
+    const wrongMeetingID = 'wrongMeetingID';
+
+    meetingsJSONAdapter.getMeeting(wrongMeetingID).subscribe(
+      () => {},
+      (error) => {
+        expect(error.message).toBe(`Could not find meeting with ID "${wrongMeetingID}"`);
```

Comment:
Should the expect be hardcoded?
e.g.
```suggestion
        expect(error.message).toBe('Could not find meeting with ID "wrongMeetingID"');
```

Reply:
No, it doesn't need to be hard coded. This matches with the consistency for all other JSON tests. Also gives us the ability to change the actual wrong meeting ID at some point if needed.